app/assets/javascripts/discourse/app/widgets/header-contents.js

```diff
@@ -15,7 +15,7 @@ createWidget("header-contents", {
       {{/if}}
     {{/if}}
 
-    {{home-logo attrs=attrs}}
+    {{home-logo-wrapper-outlet attrs=attrs}}
```

Comment:
Did you try to look in plugins/themes if someone was overriding header-contents and/or home-logo?